File: Q27/Q27.c
#include <stdio.h>

void printAlternatingPyramid(int n) {
    for (int i = 1; i <= n; i++) {
        for (int j = 1; j <= n - i; j++) {
            printf(" ");
        }

+
        char symbol = (i % 2 == 1) ? '*' : '#';
        for (int j = 1; j <= i; j++) {
            printf("%c ", symbol);
        }
        
        printf("\n");
    }
}

int main() {
    int n;
    printf("Enter the number of rows: ");
    scanf("%d", &n);
    printAlternatingPyramid(n);
    return 0;
}

File: Q27/Q27.go
package main

import (
	"fmt"
	"strings"
)

func printAlternatingPyramid(n int) {
	for i := 1; i <= n; i++ {
		fmt.Print(strings.Repeat(" ", n-i))

		var symbol string
		if i%2 == 1 {
			symbol = "*"
		} else {
			symbol = "#"
		}

		for j := 1; j <= i; j++ {
			fmt.Print(symbol + " ")
		}
		fmt.Println()
	}
}

func main() {
	var n int
	fmt.Print("Enter the number of rows: ")
	fmt.Scan(&n)
	printAlternatingPyramid(n)
}
